src/components/Wizard/Wizard.test.tsx

```diff
@@ -112,24 +113,24 @@ describe('<Wizard />', () => {
 		renderComponent(
 			{
 				file: mockFile,
-				metadata: { documentType: documentTypeEnum.typeOne, documentDescription: '', executionDate: '' },
+				metadata: { documentDescription: '', executionDate: '' },
 			},
 			'/step2',
 		);
 		fireEvent.click(screen.getByText('Opslaan'));
 		expect(onMetadataSubmitMock).not.toHaveBeenCalled();
 	});
 
-	test('Should submit when valid data', () => {
-		// not working yet
-		// renderComponent(storeState, '/step2', { formValues: mockData, isValidForm: true });
-		// // const spy = jest.spyOn(selectors, 'getFileFromStore');
-		// // expect(spy).toHaveBeenCalled();
-		// expect(screen.getByText('Vorige')).toBeInTheDocument();
-		// // console.log(container);
-		// expect(onMetadataSubmitMock).toHaveBeenCalled();
-		// fireEvent.click(screen.getByText('Opslaan'));
-		// fireEvent.click(screen.getByText('Vorige'));
-		// expect(screen.queryByTestId('step-1')).toBeInTheDocument();
-	});
+	// test('Should submit when valid data', () => {
+	// not working yet
+	// renderComponent(storeState, '/step2', { formValues: mockData, isValidForm: true });
+	// // const spy = jest.spyOn(selectors, 'getFileFromStore');
+	// // expect(spy).toHaveBeenCalled();
+	// expect(screen.getByText('Vorige')).toBeInTheDocument();
+	// // console.log(container);
+	// expect(onMetadataSubmitMock).toHaveBeenCalled();
+	// fireEvent.click(screen.getByText('Opslaan'));
+	// fireEvent.click(screen.getByText('Vorige'));
+	// expect(screen.queryByTestId('step-1')).toBeInTheDocument();
+	// });
```

Comment:
Weg of fixen, geen commentend in principe.

Reply:
Point taken. We have a ticket lined up to deal with this and missing coverage so will leave it for now.